src/test/java/wooteco/subway/admin/domain/LineTest.java

```diff
@@ -1,53 +1,54 @@
 package wooteco.subway.admin.domain;
 
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.*;
 
 import java.time.LocalTime;
 import java.util.List;
+import java.util.Objects;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 public class LineTest {
-    private Line line;
-
-    @BeforeEach
-    void setUp() {
-        line = new Line(1L, "2호선", LocalTime.of(05, 30), LocalTime.of(22, 30), 5);
-        line.addLineStation(new LineStation(null, 1L, 10, 10));
-        line.addLineStation(new LineStation(1L, 2L, 10, 10));
-        line.addLineStation(new LineStation(2L, 3L, 10, 10));
-    }
-
-    @Test
-    void addLineStation() {
-        line.addLineStation(new LineStation(null, 4L, 10, 10));
-
-        assertThat(line.getStations()).hasSize(4);
-        LineStation lineStation = line.getStations().stream()
-                .filter(it -> it.getPreStationId() == 4L)
-                .findFirst()
-                .orElseThrow(RuntimeException::new);
-        assertThat(lineStation.getStationId()).isEqualTo(1L);
-    }
-
-    @Test
-    void getLineStations() {
-        List<Long> stationIds = line.getLineStationsId();
-
-        assertThat(stationIds.size()).isEqualTo(3);
-        assertThat(stationIds.get(0)).isEqualTo(1L);
-        assertThat(stationIds.get(1)).isEqualTo(2L);
-        assertThat(stationIds.get(2)).isEqualTo(3L);
-    }
-
-    @ParameterizedTest
-    @ValueSource(longs = {1L, 2L, 3L})
-    void removeLineStation(Long stationId) {
-        line.removeLineStationById(stationId);
-
-        assertThat(line.getStations()).hasSize(2);
-    }
+	private Line line;
+
+	@BeforeEach
+	void setUp() {
+		line = new Line(1L, "2호선", LocalTime.of(05, 30), LocalTime.of(22, 30), 5, "bg-red-300");
```

Comment:
```suggestion
		line = new Line(1L, "2호선", LocalTime.of(5, 30), LocalTime.of(22, 30), 5, "bg-red-300");
```

0으로 시작하는 숫자는 8진수를 의미하는데요.
09와 같은 값을 사용한다면 의도와 다르게 동작할 수 있을 것 같습니다~

Reply:
수정했습니다!